packages/axios-extension/test/abap/abap-service-provider.test.ts

```diff
@@ -11,9 +11,16 @@ import {
 } from '../../src';
 import { ATO_CATALOG_URL_PATH } from '../../src/abap/ato';
 
-nock.disableNetConnect();
-
 describe('AbapServiceProvider', () => {
+    beforeAll(() => {
+        nock.disableNetConnect();
+    });
```

Comment:
I don't have any objections against this change, just curious, what is the advantage in running it in `beforeAll` compared to outside of the top-most describe? Similar question, do we need `enableNetConnect` in the `afterAll` call again? So far, I assumed that each test file is running separate so it won't affect the next one.